wandb/integration/ultralytics/pose_utils.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Optional
+
+import numpy as np
+from PIL import Image
```

Comment:
i guess we trusting that `ultralytics` requires these dependencies, otherwise it would make sense to check, since these are not part of the default wandb requirements

Reply:
I moved all the ultralytics-related imports inside a try-except block in callback.py.